websites/Z/ZombsRoyale.io/presence.ts

```diff
@@ -0,0 +1,108 @@
+const presence = new Presence({
+  clientId: "845354103118364672"
+});
+
+let matchStart: number = null;
+
+presence.on("UpdateData", async () => {
+  const presenceData: PresenceData = {
+    largeImageKey: "logo"
+  },
+  currentGameState = await this.getPageletiable("game.currentGameState");
```

Comment:
yeah well it isn't loool. it should be
```suggestion
  currentGameState = await getPageletiable("game.currentGameState");
```